src/transport_picoquic.cpp

```diff
@@ -246,7 +244,7 @@ pq_loop_cb(picoquic_quic_t* quic, picoquic_packet_loop_cb_enum cb_mode, void* ca
         switch (cb_mode) {
             case picoquic_packet_loop_ready: {
                 log_msg << "packet_loop_ready, waiting for packets";
-                transport->logger.log(LogLevel::info, log_msg.str());
+                transport->logger->info << log_msg.str() << std::flush;
```

Comment:
Can we drop `log_msg` and append to logger info with flush at the end?

Reply:
We could, but log_msg is populated with a prefix string above this line of code that is used in multiple case statements. Thus, it made sense to leave it as-is.